### PR TITLE
Abort tests if cleanup is not performed

### PR DESCRIPTION
**Pull request by filariow**
Signed-off-by: Francesco Ilario <filario@redhat.com>

### CODE REVIEW

File: test/acceptance/features/steps/persistent_clusters.py [hunk 1]
```diff
@@ -105,67 +107,48 @@ def cleanup_controller_namespace(self, namespace: str):
...
-        out, err = cmd.run(f"kubectl delete -n {namespace} {x} --all --ignore-not-found")
-        assert err == 0, "failed to run command!"
-
-        # delete any deployments in the namespace
-        cmd.run(f"kubectl delete -n {namespace} deployments --all --ignore-not-found --timeout=30s")
-        names, _ = cmd.run(f"kubectl get deployments.apps -o name -n {namespace}")
-        for resource in names.splitlines():
-            patch = '{"metadata": {"finalizers": []}}'
-            cmd.run(f'kubectl patch -n {namespace} {resource} -p \'{patch}\' --type=merge')
+        resources.append("deployments")
```
Comment (sadlerap) [inline test/acceptance/features/steps/persistent_clusters.py:110]: This and the other places we add `deployments` should have its group name added, e.g. "deployments.apps".

Comment (filariow) [reply test/acceptance/features/steps/persistent_clusters.py:110]: thanks, great catch

Comment (filariow) [reply test/acceptance/features/steps/persistent_clusters.py]: done

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

**Pull request merged**